server/app/services/applicant/ApplicantData.java

```diff
@@ -133,25 +133,37 @@ public Optional<String> getApplicantNameAtWellKnownPath() {
   // that we can modify both the Primary Applicant Info columns and
   // Well Known Paths at the same time.
   public Optional<String> getApplicantFirstName() {
-    return applicant.getFirstName().or(() -> readString(WellKnownPaths.APPLICANT_FIRST_NAME));
+    return applicant == null || applicant.getFirstName().isEmpty()
+        ? readString(WellKnownPaths.APPLICANT_FIRST_NAME)
```

Comment:
With this logic, will we change `readString(WellKnownPaths.APPLICANT_FIRST_NAME)` to `Optional.empty()` when well known paths go away?